### PR TITLE
fix(create-collect): add size prop to plugin-contents & add focus function to name-form

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

CollectorPluginContents Components
step1 -> sm
step2, detail -> lg

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/shared/CollectorPluginContents.vue [hunk 1]
```diff
@@ -49,15 +53,27 @@ import type { PluginReferenceItem, PluginReferenceMap } from '@/store/modules/re
...
+type Size = 'sm' | 'lg';
+
 // TODO: Add plugin data type
 interface Props {
     plugin?: CollectorPluginModel|RepositoryPluginModel|null;
+    size?: Size;
 }
 
-const props = defineProps<Props>();
+const { width } = useWindowSize();
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/shared/CollectorPluginContents.vue:64]: praise: good!

File: apps/web/src/services/asset-inventory/collector/shared/CollectorPluginContents.vue [hunk 2]
```diff
@@ -49,15 +53,27 @@ import type { PluginReferenceItem, PluginReferenceMap } from '@/store/modules/re
...
+
+const props = withDefaults(defineProps<Props>(), {
+    plugin: null,
+    size: 'lg',
+});
 
 const state = reactive({
     icon: computed<string>(() => state.pluginItem?.icon ?? ''),
+    iconSize: computed(() => {
+        if (props.size === 'sm' || width.value < 1024) return '2.5rem';
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/shared/CollectorPluginContents.vue:74]: suggestion: Please use the 'screen' object from `style.ts` file.

**Review by WANZARGEN — APPROVED** [review]
Good!! I left a suggestion~

**Review by skdud4659 — APPROVED** [review]
LGTM!!

**Pull request merged**